lib/addons/coredns/index.ts

```diff
@@ -1,15 +1,25 @@
-import { CoreAddOn } from "../core-addon";
+import { CoreAddOn, CoreAddOnProps } from "../core-addon";
+
+/**
+ * Configuration options for the coredns add-on.
+ */
+type corednsCollectorAddOnProps = CoreAddOnProps;
```

Comment:
Type naming  - must start with uppercase (classes, interfaces, types, etc.). Please see https://google.github.io/styleguide/tsguide.html for the naming conventions.